xtask/src/tasks/verify_size.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+use crate::Xtask;
+use object::read::Object;
+use object::read::ObjectSection;
+use std::collections::HashSet;
+
+/// Runs a size comparison and outputs a diff of two given binaries
+#[derive(Debug, clap::Parser)]
+#[clap(about = "Verify the size of a binary hasn't changed more than allowed.")]
+pub struct VerifySize {
+    /// Old binary path
+    #[clap(short, long, required(true))]
```

Comment:
I believe the `required(true)` is redundant, as these aren't using `Option<PathBuf>`, nor are they relying on any clap `default` directive.